modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -201,36 +202,17 @@ function VocabularyQuery(props) {
     //...Make a queue of vocabularies to search through
     setSuggestionsLoading(true);
     var vocabQueue = questionDefinition.sourceVocabularies.slice();
-    makeMultiRequest(vocabQueue, input, null, []);
+    makeMultiRequest(vocabQueue, input, [], []);
```

Comment:
Shouldn't the `statuses` parameter be an object instead of an array?
```suggestion
    makeMultiRequest(vocabQueue, input, {}, []);
```